Taarafo.Core/Brokers/Storages/StorageBroker.PostReports.References.cs

```diff
@@ -0,0 +1,21 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE TO CONNECT THE WORLD
+//
```

Comment:
We need to fix this guy with the ```-----------------------------------------```
We also need a new line between the copyrights notice and the code